azure-pipelines.yml

```diff
@@ -45,10 +45,10 @@ jobs:
     jobs:
     - job: Windows
       pool:
-        ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
           name: NetCorePublic-Int-Pool
           queue: BuildPool.Windows.10.Amd64.VS2017.Open
-        ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+        ${{ if ne(variables['System.TeamProject'], 'public') }}:
           name: NetCoreInternal-Int-Pool
           queue: BuildPool.Windows.10.Amd64.VS2017
       variables:
```

Comment:
@aspnet/build this branch lacked the following line. Is it needed in this purely Arcade / AzDO branch?
``` yaml
VSTS_OVERWRITE_TEMP: false # Workaround for https://github.com/dotnet/core-eng/issues/2812
```

Reply:
VSTS_OVERWRITE_TEMP is only needed on 2.1 and 2.2 branches which build with ProdCon.